src/main/java/org/sagebionetworks/workers/util/aws/message/PollingMessageReceiverImpl.java

```diff
@@ -110,6 +110,20 @@ public PollingMessageReceiverImpl(AmazonSQSClient amazonSQSClient,
 	 */
 	@Override
 	public void run(final ProgressCallback<Message> containerProgressCallback) throws Exception {
+		Message message = null;
+		do {
+			message = pollForMessage();

```

Comment:
This change would cause workers to take messages from the queue unthrottled until the queue is empty. Is that intended?